### PR TITLE
Add correct link to s3 plugin

### PR DESCRIPTION
**Pull request by tomcoonen**
Add correct link to s3 plugin, plugins dir is missing from master.

### CODE REVIEW

File: website/docs/plugin-storage.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ The following list of plugins are implementing the Storage API and might be used
 
 * [verdaccio-memory](https://github.com/verdaccio/verdaccio-memory) Storage plugin to host packages in Memory
 * [verdaccio-s3-storage](https://github.com/remitly/verdaccio-s3-storage) Storage plugin to host packages **Amazon S3**
-* [verdaccio-aws-s3-storage](https://github.com/verdaccio/monorepo/tree/master/plugins/aws-s3-storage) Storage plugin to host packages **Amazon S3** (maintained by Verdaccio core team)
+* [verdaccio-aws-s3-storage](https://github.com/verdaccio/monorepo/tree/9.x/plugins/aws-s3-storage) Storage plugin to host packages **Amazon S3** (maintained by Verdaccio core team)
```
[the remark this answers is not in the document]
Comment (juanpicado) [reply website/docs/plugin-storage.md]: The website still points to v5 (stable version), for context, monorepo support the 5.x branch as @edclement has referenced, while for the v6 (master) I just copied them to this repository for easy future magement (the monorepo will be archived once v6 is released and v5 expires his last days as "LTS" which are usually 6 months after next major release so people can move easily to next major. 

If we change this link to the master on this repository to the aws-plugin people will asume is stable, but is not. The aws-storage on this repo is far to be stable https://github.com/verdaccio/verdaccio/pull/2218 . So I will prefer leave it as it is, until the explained above finally happens (hopefully a long this year).

[the remark this answers is not in the document]
Comment (edclement) [reply website/docs/plugin-storage.md]: @juanpicado - given what you just stated, I think this needs to get updated to:
```markdown
[verdaccio-aws-s3-storage](https://github.com/verdaccio/monorepo/tree/verdaccio-aws-s3-storage%4010.2.1/plugins/aws-s3-storage) Storage plugin to host packages **Amazon S3** (maintained by Verdaccio core team)
```

**Comment by juanpicado** [comment]
@tomcoonen can we close this? I think was clear monorepo still is where the plugin for 5.x is hosted.

**Comment by tomcoonen** [comment]
@juanpicado I changed the link so it points to the monorepo, docs are still broken on this point so I guess this should be merged into master since 5.x branch doesn't have website dir.

**Review by juanpicado — APPROVED** [review]

**Pull request merged**